.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 | Q             | A
 | ------------- | ---
-| Branch?       | master for features / 2.7 up to 4.0 for bug fixes <!-- see below -->
+| Branch?       | master for features / 2.8 up to 4.0 for bug fixes <!-- see below -->
```

Comment:
`4.0` can be replaced with `4.1` at the same time.

Reply:
Good point. Changed 😉